examples/camera/camera_coloured_ball_detect.py

```diff
@@ -0,0 +1,44 @@
+from os import environ
+from pitopcommon.current_session_info import get_first_display
+from pitop.camera import Camera
+from pitop.processing.algorithms import BallDetector
+from signal import pause
+from pitop.processing.utils.vision_functions import import_opencv
+
+
+cv2 = import_opencv()
+
+environ["DISPLAY"] = get_first_display()
+
+
+def process_frame(frame):
+    ball = ball_detector.detect(frame, colour=("red", "green", "blue"))
```

Comment:
```suggestion
    balls = ball_detector.detect(frame, colour=("red", "green", "blue"))
```

Especially if you're going to split it out after.